app/views/PickerView.tsx

```diff
@@ -37,6 +37,9 @@ interface IItem {
 interface IPickerViewState {
 	data: IOptionsField[];
 	value: string;
+	offset: number;
+	total: number;
+	text: string;
```

Comment:
On PickerView's context, `text` is the search text, right?
Maybe we can rename it to `searchText` so it's more clear?